tests/test_wale_restore.py

```diff
@@ -92,6 +96,8 @@ def test_run(self):
             with patch.object(self.wale_restore, 'create_replica_with_s3', MagicMock(return_value=0)):
                 self.assertEqual(self.wale_restore.run(), 0)
 
+    @patch('sys.exit', MagicMock())
+    @patch.object(WALERestore, 'run', MagicMock(return_value=0))
     def test_main(self):
-        with patch('sys.exit', MagicMock(return_value=0)):
-            self.assertEqual(main(), None)
+        self.assertEqual(_main(), None)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Statement seems to have no effect](https://www.codacy.com/app/cyberdemn/patroni/file/2035720800/issues/source?bid=3127594&fileBranchId=3129207#l103)